src/fft.jl

```diff
@@ -56,7 +56,7 @@ function plan_r2r end
 ## FFT: Implement fft by calling fftw.
 
 const version = VersionNumber(split(unsafe_string(cglobal(
-    (:fftw_version,libfftw3[]), UInt8)), ['-', ' '])[2])
+    (:fftw_version,libfftw3()), UInt8)), ['-', ' '])[2])
```

Comment:
Could implement a custom struct that does what the function does in `getindex` to minimize the diff for these changes but probably not worth it.